crates/astria-bridge-withdrawer/src/withdrawer/submitter/builder.rs

```diff
@@ -0,0 +1,99 @@
+use std::{
+    sync::Arc,
+    time::Duration,
+};
+
+use astria_eyre::eyre::{
+    self,
+    ensure,
+    Context as _,
+};
+use sequencer_client::tendermint_rpc;
+use tokio_util::sync::CancellationToken;
+use tracing::{
+    info,
+    warn,
+};
+
+use super::state::State;
+
+const BATCH_QUEUE_SIZE: usize = 256;
+
+pub(crate) struct Builder {
+    pub(crate) shutdown_token: CancellationToken,
+    pub(crate) sequencer_key_path: String,
+    pub(crate) sequencer_chain_id: String,
+    pub(crate) cometbft_endpoint: String,
+    pub(crate) state: Arc<State>,
+}
+
+impl Builder {
+    /// Instantiates an `Submitter`.
+    pub(crate) async fn build(self) -> eyre::Result<(super::Submitter, super::Handle)> {
```

Comment:
1. There are multiple errors in here that don't have context
2. This constructor should ideally not be async